src/components/BooksPage/index.js

```diff
@@ -1,46 +1,29 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useSelector } from 'react-redux';
 import OneBook from './OneBook';
 import NewBook from '../NewBook';
+import NoBooksMessage from '../NoBooksMessage/index';
 import './booksPage.css';
 
 const BooksPage = () => {
-  const [data] = useState([{
-    id: 1,
-    action: 'Action',
-    title: 'The Hunger Games',
-    author: 'Suzanne Collins',
-    percent: '64',
-    chapter: 'Chapter 17',
-  }, {
-    id: 2,
-    action: 'Science Fiction',
-    title: 'Dune',
-    author: 'Frank Herbert',
-    percent: '8',
-    chapter: 'Chapter 3',
-  }, {
-    id: 3,
-    action: 'Economy',
-    title: 'Capital in the Twenty-first Century',
-    author: 'Suzanne Collins',
-    percent: '80',
-    chapter: 'Introduction',
-  }]);
+  const bookList = useSelector((state) => state.booksReducer);
 
   return (
       <>
       <div className="booksPageContainer">
 
-    {data.map((book) => (
+    {bookList.length
+      ? bookList.map((book) => (
             <OneBook oneBook={book} key={book.id} />
-    ))}
+      ))
+      : <NoBooksMessage />}
 
     <NewBook />
 
       </div>
 
       </>
```

Comment:
- [ ] I would recommend indenting your code properly so that your code looks more readable and neat.